docs/release/3.4/release-3-4-22.md

```diff
@@ -0,0 +1,234 @@
+OSG Software Release 3.4.22
+===========================
+
+**Release Date**: 2018-12-20
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   [HTCondor-CE 3.2.0](https://github.com/opensciencegrid/htcondor-ce/releases/tag/v3.2.0): [Accept host certificates for pilot jobs](https://opensciencegrid.atlassian.net/browse/SOFTWARE-3489)
+-   [frontier-squid 4.4](http://frontier.cern.ch/dist/rpms/frontier-squidRELEASE_NOTES): Update from 3.5.28
+-   [Pegasus 4.9.0](https://pegasus.isi.edu/2018/10/31/pegasus-4-9-0-released/): Update from 4.8.4
+-   HTCondor 8.6.13: [Patched memory leak when remote daemon is offline](https://htcondor-wiki.cs.wisc.edu/index.cgi/tktview?tn=6837)
+-   osg-ca-scripts 1.2.4: Drop MD5 checksum requirement, prefer SHA256
+-   stashcache-client 5.2.0: Can install stashcp locally, without using CVMFS
+-   [CVMFS 2.5.2](https://cvmfs.readthedocs.io/en/2.5/cpt-releasenotes.html):  Bug fix release
+-   osg-wn-client 3.4-5: Include stashcp on the worker-node
+-   Certificate Authority certificates based on [IGTF 1.94](http://dist.eugridpma.info/distribution/igtf/current/CHANGES)
+    -   extended validity period for the ArmeSFo CA (AM)
+    -   withdrawn expiring DFN-SLCS CA (DE)
+-   [VO Package v85](https://github.com/opensciencegrid/osg-vo-config/releases/tag/release-85)
+    -   Update `INFN` CA DN
+    -   Add backup VOMS server for `enmr.eu` and `glast.org`
+
+!!! note
```

Comment:
In addition to my comments about the announcement, let's change this to `!!! important "Important HTCondor-CE update"`